src/Showoff.jl

```diff
@@ -35,7 +35,8 @@ end
 # Floating-point
 
 function concrete_minimum(xs)
-    if done(xs, start(xs))
+    x = iterate(xs)
+    if x === nothing
```

Comment:
`isempty` should work on 0.6 also, which may be slightly better?

Reply:
Done.